### PR TITLE
BL-1036: remember and restore un-maximized position

### PR DESCRIPTION
**Pull request by phillip-hopper**

### CODE REVIEW

File: src/BloomExe/BloomExe.csproj [hunk 1]
```diff
@@ -802,14 +802,6 @@
...
     </Compile>
     <Compile Include="web\EnhancedImageServer.cs" />
-    <None Include="..\..\lib\dotnet\NDesk.DBus.dll" Condition="'$(OS)'=='Unix'">
-      <Link>NDesk.DBus.dll</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\lib\dotnet\NDesk.DBus.dll.config" Condition="'$(OS)'=='Unix'">
-      <Link>NDesk.DBus.dll.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
```
Comment (hatton) [inline src/BloomExe/BloomExe.csproj]: This probably isn't related to this commit. Ideally it would be its own commit. As it is, can you add a comment here so that we know removing ndesk was intentional?

Comment (phillip-hopper) [inline src/BloomExe/BloomExe.csproj]: Yes, I purposely removed references to files no longer used by the project.  In addition to NDesk, I also removed jquery-1.7.2, which was moved to the BloomBrowserUI project.

**Comment by hatton** [comment]
Is anything needed to handle the case where the current screen can't handle the saved bounds? E.g., a second screen is attached, you quit, then that screen is removed?

**Comment by phillip-hopper** [comment]
Maybe. I'll do some testing.

**Comment by phillip-hopper** [comment]
Testing done. Pull request updated.
